com.unity.render-pipelines.core/Tests/Editor/Volumes/VolumeComponentTests.cs

```diff
@@ -180,5 +180,66 @@ public void TestSupportedOnAvoidedIfHideInInspector()
             Assert.NotNull(volumeComponents);
             Assert.False(volumeComponents.Any());
         }
+
+        static private void TestAnimationCurveInterp(AnimationCurve lhsCurve, AnimationCurve rhsCurve, float t, float startTime, float endTime, int numSteps, float eps, bool debugPrint)
+        {
+            AnimationCurve midCurve = new AnimationCurve(lhsCurve.keys);
+            KeyframeUtility.InterpAnimationCurve(ref midCurve, rhsCurve, t);
+
+            for (int i = 0; i <= numSteps; i++)
+            {
+                float timeT = ((float)i) / ((float)numSteps);
+                float currTime = Mathf.Lerp(startTime, endTime, timeT);
+
+                float lhsVal = lhsCurve.Evaluate(currTime);
+                float rhsVal = rhsCurve.Evaluate(currTime);
+
+                float expectedVal = Mathf.Lerp(lhsVal, rhsVal, t);
+
+                float actualVal = midCurve.Evaluate(currTime);
+
+                float offset = actualVal - expectedVal;
+                if (debugPrint)
+                {
+                    Debug.Log(i.ToString() + ": " + offset.ToString());
+                }
```

Comment:
As this is code from a test, and we will need to put the parameter to true, I think it will be ok to leave the Debug.Log under a comment.

Reply:
Commenting it out.